xml/deployment_cluster_administration.xml

```diff
@@ -106,26 +106,26 @@ mail: <email address>
      ]]>
     </screen>
     <para>
-     To generate the password hash, install the <package>openldap2</package>
-     package:
-    </para>
-    <screen>&prompt.root; <command>zypper</command> <option>install openldap2</option></screen>
-    <para>
-     Then generate the password hash:
+     To generate the password hash, on a separate machine, install the 
+     <package>openldap2</package> package:
     </para>
+    <screen>
+     &prompt.root; <command>zypper</command> <option>install openldap2</option>
+    </screen>
+    <para>Then generate the password hash: </para>
     <screen>&prompt.root;<command>/usr/sbin/slappasswd</command></screen>
     <para>
-     Make sure to replace all the parameters in the template above, indicated
-     as &lt;....&gt;. The mail attribute value is used as the login to
-     &dashboard; and &kube;.
+     Make sure to replace all the parameters in the template above, indicated as &lt;....&gt;.
+     The mail attribute value is used as the login to &dashboard; and &kube;.
     </para>
     <para>
-     Populate your OpenLDAP server with this LDIF file:
+     Transfer the resulting LDIF file to your CaaSP server, and populate your 
+     OpenLDAP server with this file:
     </para>
     <screen>&prompt.root;<command>ldapadd</command> -H ldap://&lt;ADMIN NODE IP&gt;:389 -ZZ -D cn=admin,dc=infra,dc=caasp,dc=local -w &lt;LDAP ADMIN PASSWORD&gt; -f &lt;LDIF FILE&gt;</screen>
     <para>
```

Comment:
User-defined variables should use the replaceable tags rather than angle braces.

Reply:
also, check the style guide for the correct conventions for example IP addresses and host and domain names. Some of them have entities.

Reply:
Good catch. This required removing CDATA tags but I have replaced all the &gt; ... &lt; markup with <replaceable>.

Where the instructions referred to the angle brackets, I have replaced it with 

... indicated <replaceable>like this</replaceable>.

I've used entities for admin node, worker node and the product name. Those are all I could identify.